docs/config/cloud.md

```diff
@@ -48,16 +48,6 @@ exports.config = {
 }
 ```
 
-You may want to hide sensitive test data, for example authentication steps.
```

Comment:
why do you delete this?

Reply:
because we have this in the reporter already so no need for manual steps in the test.